YoCode/FileImport.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace YoCode
+{
+    // To import:
+    // UnitConverter.cs from junior-test/UnitConverterWebApp
+    // Program.cs from junior-test/UnitConverterWebApp
+    // Startup.cs from junior-test/UnitConverterWebApp
+    // HomeController.cs from junior-test/UnitConverterWebApp/Controllers
+
+    class FileImport
+    {
+        public string ORIGINAL_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\junior-test";
+        public string MODIFIED_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\original-test";
+
+        List<String> SearchPatterns = new List<string>{ "*.cs", "*.cshtml", "*.js" };
+        
+        //Will return a list of all files from a directory
+        public List<string> GetAllFilesInDirectory(String PATH)
+        {
+            List<string> files = new List<string>();
```

Comment:
There are lots of places where you could use `var` for declaring a variable (this is just one example). I would prefer it if you used var because it makes refactoring easier.